packages/base/src/util/ColorConversion.ts

```diff
@@ -196,6 +196,19 @@ const getRGBColor = (color: string): ColorRGB => {
 	return HEXToRGB(color);
 };
 
+const getAlpha = (color: string): number => {
+	let alpha = 1;
+
+	if (color.startsWith("rgba")) {
```

Comment:
we should either change it to getRGBAAlpha or we should also have hsla added to this check.

Reply:
I have added also check for hsla.